kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -396,7 +396,7 @@ function withNodes(nodeKey, callback, currentNode) {
 
 function setNodeClass(nodeKey, className) {
     withNodes(nodeKey, function(node) {
-        $(node.span).removeClass("unstarted partial complete").extraClasses(className);
+        // $(node.span).removeClass("unstarted partial complete").extraClasses(className);  no idea why run this here?
```

Comment:
This is to reset any existing classes on the nodes, so that when we set the current progress class on the nodes, then it will change appropriately.